Palo Alto Networks/paloalto-ngfw/tests/fix_bug_with_int.json

```diff
@@ -44,6 +44,7 @@
       "logger": "traffic"
     },
     "network": {
+      "application": "vsys1",
```

Comment:
It seems we have an issue with the sample. `vsys1` is the virutalLocation. According to [this documentation](https://docs.paloaltonetworks.com/pan-os/9-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/traffic-log-fields#idbe18d2d4-9eb8-4966-bec8-df3a6de70e66), the SourceUser or DestinationUser is missing from the sample.